src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventBookingManagerTest.java

```diff
@@ -934,6 +948,433 @@ void requestReservations_cancelledReservationsDoNotCountTowardsReservationLimit_
     verify(mockedObjects);
   }
 
+  @Nested
```

Comment:
a lot of this test code looks identical. Would it be worthwhile/possible refactoring it & making a parameterised method?